app/helpers/auth0_helper.rb

```diff
@@ -72,9 +72,16 @@ def auth0_session
   # (see https://auth0.com/docs/sessions/concepts/session-layers)
   def auth0_invalidate_application_session
     session.delete(:auth0_credentials)
-    warden&.logout(:auth0_user)
-    # logout users from previous Devise scope
-    warden&.logout(:user)
+    begin
+      warden.logout(:auth0_user)
+      # logout users from previous Devise scope
+      warden.logout(:user)
+    rescue
+      # if warden middleware is not present for any reason
```

Comment:
should we still log this `rescue` block?

Reply:
Yes, may be a good idea. Let me add it